app/components/Campuses.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import store from '../store.jsx'
+import Navbar from './Navbar'
+import { NavLink } from 'react-router-dom'
+
+
+
+const mapStateToProps = function(state) {
```

Comment:
I like that you've put `mapStateToProps` and `mapDispatchToProps` at the top of the file. As a reader it gives me a heads-up for the props I'm about to see in the component below.